### ISSUE TITLE
Options for Icon Color? (differentiate from Mailbox and other blue apps)

### ISSUE THREAD

**Issue by SimplGy** [uranusjr/macdown#334]
This is crazy, and probably only me, but I always have `MacDown` and `Mailbox` open. They both have electric blue icons with white insignias in them surrounded by white rings, and both start with the letter "M". I always have difficulty when `cmd`+`tab`ing remembering which is which.

![screen shot 2015-04-05 at 2 44 52 pm](https://cloud.githubusercontent.com/assets/548809/6996059/650ca80e-dba2-11e4-9b17-26ea84270d91.png)

Maybe nobody else has this problem. It's obscure. It only happens with these two apps together. It involves a failure of all the mnemonic devices particular to my brain. It's such a corner case and so ridiculous, but it drives me crazy every day.

I'm willing to do the different icon color options and add a little `blue/green/orange` option to preferences.

Maybe other folks would think it's cool to have different colored icons.

Would you be open to a PR for this?

**Comment by SimplGy** [comment]
If anyone else has this "issue" and wants to self medicate, this does the trick:
https://dribbble.com/shots/1619305-MacDown-Icon-replacement

Just "view package contents" in MacDown.app and replace the icon file

![screen shot 2015-04-05 at 3 10 30 pm](https://cloud.githubusercontent.com/assets/548809/6996111/f40d7b20-dba5-11e4-8912-ab1e3629ee2c.png)

**Comment by uranusjr** [comment]
Probably too specific a problem, and I’d rather focus my (already too little) time working on other things. This is very doable by setting `NSApplication`’s `applicationIconImage` property, though. I’d be more than happy to accept a pull request if anyone’s interested in implementing it.

Leaving this open so that someone might be interested.

**Comment by SimplGy** [comment]
Oh ya fair enough. Thanks for giving this issue more attention than it deserves :)

I think it might be fun to try to do this, actually. I'll take a crack at it if I get a free day.

**Comment by alexkaessner** [comment]
Good idea. This reminds me of [Adium](https://adium.im) or [xScope](http://xscopeapp.com), where you can choose you app icon (color).

If you need help with the app icon recoloring I could help or share [my icon from dribbble](http://drbl.in/lASz)! :wink:
